Puppetfile

```diff
@@ -1,26 +1,18 @@
 forge 'https://forgeapi.puppetlabs.com/'
 
-# Dependencies
-# For Puppet 7 support
-mod 'puppetlabs/xinetd',             :git => 'https://github.com/puppetlabs/puppetlabs-xinetd', :commit => 'e742608dccdf42236144acf9f05e483b47c576f1'
-mod 'puppetlabs/postgresql',         '>= 7.0.0'
-mod 'theforeman/dhcp',               :git => 'https://github.com/theforeman/puppet-dhcp'
-mod 'theforeman/dns',                :git => 'https://github.com/theforeman/puppet-dns'
-mod 'theforeman/git',                :git => 'https://github.com/theforeman/puppet-git'
-mod 'theforeman/puppetserver_foreman', :git => 'https://github.com/theforeman/puppet-puppetserver_foreman'
-mod 'theforeman/tftp',               :git => 'https://github.com/theforeman/puppet-tftp'
-
-# Katello dependencies
-mod 'katello/candlepin',             :git => 'https://github.com/theforeman/puppet-candlepin'
-mod 'theforeman/pulpcore',           :git => 'https://github.com/theforeman/puppet-pulpcore'
-mod 'katello/qpid',                  :git => 'https://github.com/theforeman/puppet-qpid'
-
-# Top-level modules
-mod 'theforeman/foreman',            :git => 'https://github.com/theforeman/puppet-foreman'
-mod 'theforeman/foreman_proxy',      :git => 'https://github.com/theforeman/puppet-foreman_proxy'
-mod 'theforeman/puppet',             :git => 'https://github.com/theforeman/puppet-puppet'
-
-# Top-level katello modules
-mod 'katello/foreman_proxy_content', :git => 'https://github.com/theforeman/puppet-foreman_proxy_content'
-mod 'katello/certs',                 :git => 'https://github.com/theforeman/puppet-certs'
-mod 'katello/katello',               :git => 'https://github.com/theforeman/puppet-katello'
+mod 'puppetlabs/xinetd',               '>= 3.3.0 < 3.4.0'
```

Comment:
This one must remain on git since there is no release that's Puppet 7 compatible. Yes, it is ironic that something in the puppetlabs namespace is the last to add Puppet 7 compatibility, but that's the way it is sadly.

Reply:
🤦🏼 updating